docs/ru/assemble_4.md

```diff
@@ -0,0 +1,336 @@
+# Сборка Клевера 4
+
+<img src="../assets/clever-4-blow.png" class="zoom center" width=400>
+
+## Сборка основы для рамы
+
+1. Совместите 2 карбоновые пластины жесткости, используя центральные пазы.
+2. Используя пазы, установите сверху карбоновую центральную деку.
+
+    <center>
+        <img src="../assets/4/01.png" width=300 class="zoom border">
+        <img src="../assets/4/02.png" width=300 class="zoom border">
+    </center>
+
+3. Стяните конструкцию с помощью винтов М3х8 и стальных гаек, установленных в пазах пластин.
+
+    <center>
+        <img src="../assets/4/03.png" width=300 class="zoom border">
+        <img src="../assets/4/03_1.png" width=300 class="zoom border">
+    </center>
+
+## Установка моторов
+
+1. Распакуйте моторы.
+2. Используя ножницы, укоротите провода на моторах:
+
+    * Обрежьте половину длины (оставив 30 мм).
+    * Зачистите (снимите 5 мм изоляции с конца провода, не повредив медные жилы).
+
+    <img src="../assets/4/04.png" width=300 class="zoom center border">
+
+    * Скрутить провода.
+    * [Залудить провода](tinning.md), используя пинцет.
+
+3. Установите мотор на луч.
+4. Прикрепите мотор к лучу винтами М3х5, используя шестигранный ключ.
```

Comment:
I'm being pedantic here, but we should probably say that the user has to perform these actions for each of the motors.

Reply:
OK, fix it at your taste.